db/migrations/20180719130843_boek_fields.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+use Phinx\Migration\AbstractMigration;
+
+class BoekFields extends AbstractMigration
+{
+    public function change()
```

Comment:
`public function up()`
Deze functie is niet reversable. Een lege `down` functie is dan denk ik ook nodig.